ibm/service/secretsmanager/resource_ibm_sm_service_credentials_secret.go

```diff
@@ -276,9 +276,14 @@ func ResourceIbmSmServiceCredentialsSecret() *schema.Resource {
 			},
 			"ttl": &schema.Schema{
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
ttl  is moved from required to optional
If user don't have any ttl will it comes with any default value if so we need to make both optional and computed else it will find a dif in next plan

Reply:
Add a testcase scenarion for this